dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/MetadataFieldRestRepository.java

```diff
@@ -98,4 +113,116 @@ public MetadataFieldRest findOne(Context context, Integer id) {
     public MetadataFieldResource wrapResource(MetadataFieldRest bs, String... rels) {
         return new MetadataFieldResource(bs, utils, rels);
     }
+
+    @Override
+    @PreAuthorize("hasAuthority('ADMIN')")
+    protected MetadataFieldRest createAndReturn(Context context)
+            throws AuthorizeException, SQLException {
+
+        // parse request body
+
+        MetadataFieldRest metadataFieldRest;
```

Comment:
Thanks for the inline comments!  Just a minor thing, but the empty line after each of the inline comments is not necessary (I noticed that occurs throughout this method).